dev/com.ibm.ws.cdi.ejb_fat/fat/src/com/ibm/ws/cdi/ejb/apps/newTimer/TestEjbTimerServlet.java

```diff
@@ -0,0 +1,72 @@
+ /*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cdi.ejb.apps.newtimer;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+
+import javax.ejb.EJB;
+import javax.servlet.ServletException;
+import javax.servlet.ServletOutputStream;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import componenttest.app.FATServlet;
+import org.junit.Test;
+
+@WebServlet("/Timer")
+public class TestEjbTimerServlet extends FATServlet {
+    private static final long serialVersionUID = 1L;
+    @EJB
+    EjbSessionBeanLocal bean;
+
+    private boolean needToSetUp = true;
+
+    private synchronized void setUp() {
+        if (needToSetUp) {
+            bean.initTimer();
+            needToSetUp = false;
+            try {
+                Thread.sleep(3000);
+            } catch (InterruptedException e) {
+                assertTrue("The timer exploded", false);
+            }
+        }
+    }
+
+    @Test
+    public void testRequestScopeActiveWithinTimeout() throws ServletException, IOException {
+        setUp();
+        assertTrue("The request scope was not active inside a timeout method", EjbSessionBean.canAccessRequestScope);
```

Comment:
Because the field is a `Boolean` rather than a `boolean`, I think using `assertTrue` will result in a `NullPointerException` during autoboxing if the field is `null`.

Using `assertEquals(Boolean.TRUE, MyClass.myField);` here would be better as that will handle the `null` case.